.github/workflows/ci.yml

```diff
@@ -14,4 +18,4 @@ jobs:
       - name: Pull CI container image
         run: ./.ci/run-container-ci pull
       - name: Run CI in container
-        run: ./.ci/run-container-ci ${{github.workspace}} ${{github.base_ref}}
```

Comment:
github.base_ref is only available when the event type is pull request. Therefore we remove it.